utils/api.py

```diff
@@ -808,7 +822,7 @@ def soup_parse(soup):
 
     async def get_placement(self, username):
         resp = await _query_api(SITE_URL + f'user/{username}', 'text')
-        soup = BeautifulSoup(resp, features="html5lib")
+        soup = BeautifulSoup(resp, features="html.parser")
```

Comment:
Why html.parser?

Reply:
I kept getting `bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: html5lib. Do you need to install a parser library?` and read you should do this at coursera-dl/edx-dl#434

Reply:
Perhaps check if you have the html5lib package?

Reply:
wait that worked lol

Reply:
Please revert back to html5lib <3